hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/TestBlockDeletingService.java

```diff
@@ -236,6 +241,26 @@ private void createPendingDeleteBlocksSchema2(int numOfBlocksPerContainer,
       int txnID, long containerID, int numOfChunksPerBlock, ChunkBuffer buffer,
       ChunkManager chunkManager, KeyValueContainer container,
       KeyValueContainerData data) {
+    createPendingDeleteBlocksViaTxn(numOfBlocksPerContainer, txnID,
+        containerID, numOfChunksPerBlock, buffer, chunkManager,
+        container, data);
+  }
+
+  @SuppressWarnings("checkstyle:parameternumber")
+  private void createPendingDeleteBlocksSchema3(int numOfBlocksPerContainer,
```

Comment:
Looks like createPendingDeleteBlocksSchema3 and createPendingDeleteBlocksSchema2 are identical.

Reply:
Yes, it is, here I just intend to keep the schema check style of function in `createToDeleteBlocks` aligned, but I could remove one of the identical functions and use the lower level function directly when we have schema v2 || v3.